playbooks/roles/deploy-osh/tasks/component-runtime-setup.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: "Generate suse override for {{ override.suse }}"
+  template:
+    src: "{{ override.suse }}.j2"
+    dest: "/tmp/socok8s-suse-{{ override.suse }}"
+
+- name: Generate user overrides
+  copy:
+    content: "{{ override.user | to_nice_yaml }}"
+    dest: "/tmp/socok8s-user-{{ override.suse }}"
```

Comment:
Also seen here: it would make more sense if the variable would be named file/path

Reply:
Done.